### ISSUE TITLE
Staging Review finding: Add additional information to date of termination guidance 

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#135397]
_Need help? Please review [how to read a Staging Review ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/anatomy-of-a-staging-review-issue-ticket). Tag `@platform-governance-team-members` on Slack if you need further assistance._

## Product Information
**Team:** GovCIO VEBT
**Product:** Authorization To Disclose Personal Information To a Third Party - Education Benefits
**Feature:** VA Form 22-10278

## Findings details

**VA.gov Experience Standard - issue:** [User encounters design components or patterns that are inconsistent or confusing.](https://design.va.gov/about/experience-standards/#user-encounters-consistent-clear-design-components)
**VA.gov Experience Standard - category:** [Consistency](https://design.va.gov/about/experience-standards/#category-consistency)

**Launch-blocking:** No 
**Design System review:** No
**Collab Cycle Reviewer:** @kkitagawa-bosch (content), @jasonday (Accessibility), @allison0034 (Design)

### Description

When asking the user to provide a date of termination, the explanatory text says to pick a date, but doesn't specify any rules aroud that date. If a user then enters a date more than five years in the future, they receive an error message, which can be frustrating when they weren't given the information up front to avoid that error. 

<details><summary>Link, screenshot or steps to recreate</summary>

<img width="990" height="590" alt="Image" src="https://github.com/user-attachments/assets/cbabd732-1e22-49ca-83a6-cb201eb47a60" />

</details>

## Recommended action

Add explanatory text to warn users that the date they select must be within five years of today's date. 

## References

- **Accessibility Defect Severity:** N/A
- **WCAG Success Criteria:** N/A
- **Modality:** N/A
- **Design System Component:** N/A
- **Design System Pattern or Template:** N/A
- **Design System Foundation:** N/A
- **Context:** N/A
- **VADS Guidance:** N/A

---

### Next Steps for the VFS Team

_How did this touchpoint go? [Give feedback on the Collaboration Cycle](https://ows.io/qs/o3jkwoe) at any time._
- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members` with any questions or to get help validating the issue.
- [ ] **Ticket ownership:** If you believe that this issue is out of scope, not your team's responsibility, or a Design System issue; or if you believe this finding should not be fixed; comment and tag `@platform-governance-team-members` on your team channel in Slack to provide an explanation and who you believe is responsible. The Governance team will follow up.
- [ ] **Close this ticket** when the issue has been resolved or validated by your Product Owner.
